Presentation.Web/app/components/local-config/ui-customization/local-ui-customization.view.html

```diff
@@ -38,14 +38,32 @@
                 </td>
                 <td ng-if="tabConfig.children.length > 0">
                     <div ng-repeat="fieldConfig in tabConfig.children">
-                        <label class="local-ui-customization-option">
-                            <input id="{{::fieldConfig.key}}" type="checkbox" ng-checked="fieldConfig.available" ng-disabled="!fieldConfig.editable" ng-click="ctrl.toggleSetting(fieldConfig.key)" />
-                            {{:: fieldConfig.text}}
-                            <span ng-if="fieldConfig.helpText" class="help-block small">
-                                <i class="fa fa-info-circle" aria-hidden="true"></i>
-                                {{:: fieldConfig.helpText}}
-                            </span>
-                        </label>
+                        <table class="full-width">
+                            <tr>
+                                <td width="50%" style="vertical-align: top;">
+                                    <label class="local-ui-customization-option">
+                                        <input id="{{::fieldConfig.key}}" type="checkbox" ng-checked="fieldConfig.available" ng-disabled="!fieldConfig.editable" ng-click="ctrl.toggleSetting(fieldConfig.key)"/>
+                                        {{:: fieldConfig.text}}
+                                        <span ng-if="fieldConfig.helpText" class="help-block small">
+                                            <i class="fa fa-info-circle" aria-hidden="true"></i>
+                                            {{:: fieldConfig.helpText}}
+                                        </span>
+                                    </label>
+                                </td>
+                                <td width="50%" ng-if="fieldConfig.children.length > 0">
```

Comment:
the additional level was not required. A "group" setting is a single setting covering multiple fields so the user should only see one checkbox and the help text could then mention which fields it covers